src/components/reformed.js

```diff
@@ -51,6 +53,24 @@ const makeWrapper = WrappedComponent => {
       this.setState({ submitted });
     }
 
+    onModelSavedToServer() {
+      this.setState(
+        prevState => ({
+          initialModel: prevState.model,
+          savedToServer: true,
+        }),
+        () => {
+          // set all fields to dirty = false
+          Object.keys(this.state.fields).forEach(field => {
```

Comment:
https://github.com/NDLANO/editorial-frontend/pull/369/files medfører at setModel blir kalt ved lagring. Det holder da kanskje å endre den til å kalle `this.setState({ model, initialModel: model, fields: {} });`.

Reply:
Hmm, du mener kanskje at det holder å endre til bare å resette fields? For det stemmer